docs/source/references/config_files/project/base_yml.rst

```diff
@@ -258,6 +328,18 @@ this.
       port: 5672
       user: omniport_user
       password: omniport_password
+  integrations:
```

Comment:
Omit this as this anyway an optional configuration.

Or I think it does throw an error when `integrations` is left blank. Find out why and open an issue about it. You may also submit a PR then.

Reply:
Don't omit this; this is not an optional field but can be considered as a required idea. An idea that how can one integrate 3rd party apps configuration through configuration files. If you think of any word can be used instead of integrations otherwise as it is finally parsed and used in the app on convenience. We have already configured sentry in the omniport accordingly to find if `integrations` key is there and have a sentry object. I would suggest to add this key with by default an empty dictionary as written in `base_stemcil.yml` for the sake of the propagation of this idea.